tests/testAddToCoadd.py

```diff
@@ -44,19 +44,16 @@
     display
 except NameError:
     display = False
-    Verbosity = 0 # increase to see trace
+    Verbosity = 0  # increase to see trace
 
 pexLog.Trace_setVerbosity("lsst.coadd.utils", Verbosity)
 
 try:
     dataDir = lsst.utils.getPackageDir('afwdata')
+    medMIPath = os.path.join(dataDir, "data", "med.fits")
```

Comment:
I'll make the general comment that the tests use `medMIPath` but skip on `dataDir`. The tests themselves don't use `dataDir` so I think I'd prefer it if `medMIPath` was the thing that was set to `None` and was the thing that the skip test used. It seems more consistent to me to skip on the thing that the test is using.